src/python/director/ikconstraintencoder.py

```diff
@@ -43,25 +38,3 @@ def getPlanPoses(constraints, ikPlanner):
     poses = sorted([c.postureName for c in constraints if hasattr(c, 'postureName')])
     poses = {poseName:list(ikPlanner.jointController.getPose(poseName)) for poseName in poses}
     return poses
-
-
-class IKConstraintEncoder(object):
-    def __init__(self,ikPlanner):
-        self.ikPlanner = ikPlanner
-
-    def publishConstraints(self,constraints,messageName='PLANNER_REQUEST'):
```

Comment:
We do not need this as we are handling constraint encoding for EXOTica [here](https://github.com/RobotLocomotion/director/blob/ce215f77ec1aa104e86b56f2b0f0363c219fa691/src/python/director/plannerPublisher.py#L39) using the ``getPlanPoses`` method which remains untouched